api/api.go

```diff
@@ -270,6 +270,10 @@ func (c *Client) Status(trackID string) (*StatusInfo, error) {
 
 	defer res.Body.Close()
 
+	if _, err := c.List(trackID); err != nil {
+		return nil, err
+	}
+
```

Comment:
No, we just need to return an error here if the Track doesn't exist. 

I looked into the API a bit and it looks like the `/tracks/<trackID>/status` endpoint returns the same response for both valid and invalid tracks, so we just need to first check that the track exists and return an error if it doesn't instead of parsing the response.

If there's no error here, then `Status` looks at the fetched problems and recent problems, so we don't need the `List` results.